src/offsets.cpp

```diff
@@ -28,6 +28,8 @@ void Offsets::GetOffsets()
 	offsets.DT_BaseEntity.m_MoveType = offsets.DT_BaseEntity.m_nRenderMode + 1;
 	offsets.DT_BaseEntity.m_Collision = NetVarManager::GetOffset(tables, "DT_BaseEntity", "m_Collision");
 	offsets.DT_BaseEntity.m_bSpotted = NetVarManager::GetOffset(tables, "DT_BaseEntity", "m_bSpotted");
+	offsets.DT_BaseEntity.m_flSimulationTime = NetVarManager::GetOffset(tables, "DT_BaseEntity", "m_flAnimTime");
```

Comment:
You're late https://github.com/AimTuxOfficial/AimTux/commit/247f460fc2a5bce7cf644c2d5249f630af283ed6